qiskit/circuit/library/standard_gates/equivalence_library.py

```diff
@@ -100,6 +104,32 @@
             def_ms.append(RXXGate(theta), [q[i], q[j]])
     _sel.add_equivalence(MSGate(num_qubits, theta), def_ms)
 
+# PhaseGate
+
+q = QuantumRegister(1, 'q')
+theta = Parameter('theta')
+phase_to_u1 = QuantumCircuit(q)
+phase_to_u1.u1(theta, 0)
+_sel.add_equivalence(PhaseGate(theta), phase_to_u1)
```

Comment:
Can you also add the reverse mapping `u1_to_phase`? (Same for `u3_to_u`)

Reply:
done in 3140040